mission-vision-values.md

```diff
@@ -0,0 +1,99 @@
+# OTel Mission, Vision, and Values
+
+## Mission (our overall north star as a community)
+
+OpenTelemetry's Mission: **to enable effective observability through
+high-quality, turnkey, open-source telemetry.**
+
+## Vision (the world we imagine for OTel end-users)
+
+Excellent observability requires high-quality telemetry, as well as the
+performant, consistent instrumentation that makes it possible.
+
+Telemetry in this context is the firehose of raw observational data streaming
+out of software applications, and while “high-quality telemetry” may be a
+requirement for excellent observability, it’s still unreasonable and
+unrealistic to expect software developers to add or maintain the necessary
```

Comment:
software developers in this case are not library owners. Maybe this needs to be highlighted somehow.

Reply:
by library I mean client libraries that needs to be instrumented with OTel

Reply:
need to distinguish these software developers and end users

Reply:
I made this "developers of application software" – good call-out.

Reply:
I like "developers of application software".